packages/gatsby-theme-newrelic/src/components/GlobalNavLink.js

```diff
@@ -19,7 +20,12 @@ const GlobalNavLink = ({ children, href }) => {
     }
   `);
 
-  const isCurrentSite = href.startsWith(siteUrl);
+  const location = useLocation();
+  let isCurrentSite = href.startsWith(siteUrl);
+
+  if (location.pathname.includes('/instant-observability/')) {
+    isCurrentSite = href.includes('/instant-observability/');
+  }
```

Comment:
If we're going to hard-code the path here (and I recommend we don't, see below), then we should opt for a ternary for `isCurrentSite` rather than a `let` we modify - it will help our code be more safe.

Furthermore, I'm not sure how I feel about hard-coding this logic. I think we might want to consider adopting the approach of [open for extension, but closed for modification](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle). If we modify this code and then we find we need to add _another_ site that needs to have a custom highlight, then we will need to modify this code again.

A way we could _extend_ this would be to add an optional prop to the parent component specifying which NavLink should be considered active:

```jsx
// where the component is called
<GlobalHeader active="https://newrelic.com/instant-observability" />

// in the parent component
<GlobalNavLink /* ..other props*/ active={active} />

// in this component
const isCurrentSite = href === active || (!href && href.startsWith(siteUrl);
```

Another way could be to handle it on the developer site itself using css. Something like this might work:

```css
a {
  /* reset styles */
}

a[href="https://newrelic.com/instant-observability"] {
  /* active state */
}
```

Reply:
I put up [an alternative PR](https://github.com/newrelic/gatsby-theme-newrelic/pull/453) with these ideas. Let me know if you like one over the other.

Reply:
Sorry meant to respond here yesterday, wanted to talk about this approach to see if it require changes to all sites

Reply:
I do prefer the alternative approach and it sets us up better because we don't have to hardcode, I think Jerel is reviewing the other PR but i'll take a look as well. My preference would be the alternative anywho